### ISSUE TITLE
Block p2p/proxy/VPN/anonymizer websites with family protection

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdguardDNS#101]
Source:
https://forum.adguard.com/index.php?threads/adguard-dns-discuss.18080/

**Comment by sandybeachit** [comment]
This would be good to add to the family option or perhaps as separate categories?

Such as we create an account on nadguard then we can choose what to block so we block all P2P, proxy etc or just one category.

**Comment by ameshkov** [comment]
Yeah, it requires us to have user accounts, just like what OpenDNS provides. We will do it in the future, maybe with some additional paid features.

**Comment by evgeniy-ADG** [comment]
https://github.com/AdguardTeam/AdguardForWindows/issues/887

**Comment by ArchiveBase** [comment]
Vote for Subj.

**Comment by ArchiveBase** [comment]
https://github.com/AdguardTeam/AdGuardHome/issues/951 https://github.com/AdguardTeam/AdGuardHome/issues/962

**Comment by Omoeba** [comment]
how would you block a vpn using dns though?
